src/participant/sampleDocuments.ts

```diff
@@ -59,11 +58,14 @@ export async function getStringifiedSampleDocuments({
   }
 
   const stringifiedDocuments = toJSString(additionToPrompt);
-  promptInputTokens = await model.countTokens(prompt + stringifiedDocuments);
+
+  // TODO: model.countTokens will sometimes return undefined - at least in tests. We should investigate why.
```

Comment:
This was something Alena ran into, `The model returned by vscode.lm.selectChatModels is always undefined in tests`. So there's no model to count the tokens.